src/Sylius/Behat/Element/Admin/Crud/FormElement.php

```diff
@@ -34,6 +34,15 @@ public function getValidationMessage(string $element, array $parameters = []): s
         return $validationMessage->getText();
     }
 
+    protected function waitForFormUpdate(): void
+    {
+        $form = $this->getElement('form');
+        sleep(1); // we need to sleep, as sometimes the check below is executed faster than the form sets the busy attribute
```

Comment:
Maybe it will be enough:
```suggestion
        usleep(500000); // we need to sleep, as sometimes the check below is executed faster than the form sets the busy attribute
```